Artifacts/MsSql/Structure/Admin/0160-Rename-OdsInstanceContext.sql

```diff
@@ -0,0 +1,41 @@
+-- SPDX-License-Identifier: Apache-2.0
+-- Licensed to the Ed-Fi Alliance under one or more agreements.
+-- The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+-- See the LICENSE and NOTICES files in the project root for more information.
+
+ALTER TABLE [dbo].[OdsInstanceContext]
+DROP CONSTRAINT [FK_OdsInstanceContext_OdsInstanceId_OdsInstanceId];
```

Comment:
Before trying to drop Constraint, Please check Exists or not , Then if exists drop Constraint